app/controllers/concerns/course_basic_detail_concern.rb

```diff
@@ -89,18 +104,73 @@ def build_previous_course_creation_params
   end
 
   def next_step(current_step:)
-    if current_step == :outcome
+    next_step = NextCourseCreationStepService.new.execute(current_step: current_step)
+    next_page = course_creation_path_for(next_step)
+
+    if next_page.nil?
+      raise "No path defined for next step: #{next_step}"
+    end
+
+    next_page
+  end
+
+  def course_creation_path_for(page)
+    case page
+    when :apprenticeship
+      new_provider_recruitment_cycle_courses_apprenticeship_path(
+        params[:provider_code],
+        params[:recruitment_cycle_year],
+        course: course_params,
+      )
+      # Currently the page isnt built - so skip
```

Comment:
What does this comment mean, btw?

Reply:
I should expand that a bit - currently the `sites/new` page doesn't exist - so we skip that step to allow it to get to the confirmation page as is